jwst/datamodels/ramp.py

```diff
@@ -45,8 +43,9 @@ def __init__(self, init=None, **kwargs):
         self.err = self.err
 
 
-def MIRIRampModel(*args, **kwargs):
-    warnings.simplefilter('default')
-    warnings.warn(message="MIRIRampModel is deprecated and will be removed.  "
-        "Use RampModel.", category=DeprecationWarning)
-    return RampModel(*args, **kwargs)
+@deprecate_class(RampModel)
+class MIRIRampModel:
+    """A data model for 4D MIRI ramps.
+
+    This model has been deprecated. Please use `RampModel` instead.
```

Comment:
It appears that the old function would automatically load the offending input into a `RampModel` and return it. What happens with the new class? Does it just issue a warning and return nothing?